<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewScenarioDialog</class>
 <widget class="QDialog" name="NewScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать новый сценарий</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-size: 9pt;
}

QPushButton
{
	background-color: rgb(218, 218, 218);
	border: none;
	border-radius: 6px;
	min-width: 80px;
	min-height: 30px;
	font-size: 8pt;
}

QPushButton:hover
{
	border-top: 1px solid black;
	border-left: 1px solid black;
	border-right: 2px outset black;
	border-bottom: 2px outset black;
}

QPushButton:pressed
{
	border: 2px inset black;
}

QLineEdit,
QComboBox
{
	border: 1px solid black;
	border-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leScenarioName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Область действия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbScenarioScope"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewScenarioDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewScenarioDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
